definitions/variable/economy/economy.yaml

```diff
@@ -212,4 +212,38 @@
     unit: [EUR_2020/t CO2, USD_2010/t CO2]
     skip-region-aggregation: true
 
-
+- Extraction Limit:
```

Comment:
Does it make sense to have one limit across all fossil resources? Shouldn't this be distinguished between different fuels?

Reply:
In general, it is true. Nevertheless, we have used a single parameter to control the extraction of oil and gas. I can make three parameters: one for oil and one for gas and use the same parameter in the data upload, though. I will make a new pull request with the updated variables

Reply:
No, do NOT make a new pull request - this is not how GitHub works.

Reply:
Ok. I only updated the description of the variables, then.